src/api/guardian.js

```diff
@@ -1,5 +1,6 @@
 const Guardian = require("guardian-js");
 const guardian_filter = require("./guardian_filter");
+
 const guardianKey = (query, cb) => {
```

Comment:
yeah this is weird lets change it to something else

Reply:
@Karyum do you have a suggestion of what might be better?

Reply:
@Karyum @harrygfox  We changed it into guardianListing